### PR TITLE
feat(event): guest event detail

### PR DESCRIPTION
**Pull request by mthinh**
# What ?
- Guest event detail ui

# Screenshot 
Screenshot 2023-09-05 at 14 52 56
Screenshot 2023-09-05 at 14 52 58

### CODE REVIEW

File: lib/core/presentation/pages/event/guest_event_detail_page/widgets/guest_event_detail_appbar.dart [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        background: _EventDetailCover(event: event),
+      ),
+    );
+  }
+}
+
+
+class _EventDetailCover extends StatelessWidget {
+  const _EventDetailCover({required this.event});
+
```
Comment (roncoderpush) [inline lib/core/presentation/pages/event/guest_event_detail_page/widgets/guest_event_detail_appbar.dart:54]: Why we have to separate this, can't it be merge to 1 class only ?

File: lib/core/presentation/pages/event/guest_event_detail_page/widgets/guest_event_detail_hosts.dart [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      mainAxisSize: MainAxisSize.min,
+      children: [
+        Padding(
+          padding: EdgeInsets.symmetric(horizontal: Spacing.smMedium),
+          child: Text(
+            StringUtils.capitalize(t.common.host(n: 2)),
+            style: TextStyle(
+              fontSize: 16.sp,
```
Comment (roncoderpush) [inline lib/core/presentation/pages/event/guest_event_detail_page/widgets/guest_event_detail_hosts.dart:42]: use Typo class

File: lib/core/presentation/pages/event/guest_event_detail_page/widgets/guest_event_detail_photos.dart [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+      mainAxisSize: MainAxisSize.min,
+      children: [
+        Padding(
+          padding: EdgeInsets.symmetric(horizontal: Spacing.smMedium),
+          child: Text(
+            StringUtils.capitalize(t.common.photo(n: 2)),
+            style: TextStyle(
+              fontSize: 16.sp,
+              fontWeight: FontWeight.w600,
+              color: colorScheme.onPrimary,
```
Comment (roncoderpush) [inline lib/core/presentation/pages/event/guest_event_detail_page/widgets/guest_event_detail_photos.dart:39]: Use Typo class

File: lib/core/presentation/pages/event/guest_event_detail_page/widgets/guest_event_location.dart [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+class GuestEventLocation extends StatefulWidget {
+  const GuestEventLocation({
+    super.key,
+    required this.event,
+  });
+
+  final Event event;
+  // TODO
+  // static const CameraPosition _kGooglePlex = CameraPosition(
```
Comment (roncoderpush) [inline lib/core/presentation/pages/event/guest_event_detail_page/widgets/guest_event_location.dart:20]: Whether you leave a TODO some more information, or delete uncomment codes. This action keep the project clean and informative

File: lib/core/presentation/pages/event/guest_event_detail_page/widgets/guest_event_location.dart [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  Assets.icons.icLock.svg(),
+                  const Spacer(),
+                  Text(
+                    t.event.eventLocation,
+                    style: TextStyle(
+                      fontSize: 16.sp,
+                      fontFamily: FontFamily.switzerVariable,
```
Comment (roncoderpush) [inline lib/core/presentation/pages/event/guest_event_detail_page/widgets/guest_event_location.dart:99]: Use Typo

File: lib/core/presentation/pages/event/guest_event_detail_page/widgets/more_less_description.dart [hunk 6]
```diff
@@ -0,0 +1,90 @@
+import 'package:app/gen/fonts.gen.dart';
+import 'package:app/i18n/i18n.g.dart';
+import 'package:app/theme/color.dart';
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+
+class MoreLessDescription extends StatefulWidget {
+  const MoreLessDescription({
+    super.key,
```
Comment (roncoderpush) [inline lib/core/presentation/pages/event/guest_event_detail_page/widgets/more_less_description.dart:10]: Consider using ExpandableNotifier for better animation when the text expanding or collapsing

**Review by roncoderpush — DISMISSED** [review]

File: lib/core/presentation/pages/event/guest_event_detail_page/widgets/guest_event_detail_appbar.dart (hunk 1 above)
Comment (mthinh) [reply lib/core/presentation/pages/event/guest_event_detail_page/widgets/guest_event_detail_appbar.dart:54]: @roncoderpush 
The event's cover is part of event sliver app bar, I want to separate for easier to look and separate for single concern

File: lib/core/presentation/pages/event/guest_event_detail_page/widgets/more_less_description.dart (hunk 6 above)
Comment (mthinh) [reply lib/core/presentation/pages/event/guest_event_detail_page/widgets/more_less_description.dart]: 👍

**Review by sieuhuflit — APPROVED** [review]

**Pull request merged**